stdlib/2and3/cgi.pyi

```diff
@@ -7,7 +7,10 @@ def parse(fp: IO[Any] = ..., environ: Mapping[str, str] = ...,
           keep_blank_values: bool = ..., strict_parsing: bool = ...) -> Dict[str, List[str]]: ...
 def parse_qs(qs: str, keep_blank_values: bool = ..., strict_parsing: bool = ...) -> Dict[str, List[str]]: ...
 def parse_qsl(qs: str, keep_blank_values: bool = ..., strict_parsing: bool = ...) -> Dict[str, List[str]]: ...
-def parse_multipart(fp: IO[Any], pdict: Mapping[str, bytes]) -> Dict[str, List[bytes]]: ...
+if sys.version_info >= (3, 7):
+    def parse_multipart(fp: IO[Any], pdict: Mapping[str, bytes], encoding: str, errors: str) -> Dict[str, List[Union[str, bytes]]]: ...
```

Comment:
`encoding` and `errors` have default arguments, so they should get a `= ...`.

Also, the return value should be `Dict[str, List[Any]]`, unfortunately. `List[Any]` is returned by `FieldStorage.getlist()` (which is called by `parse_multipart()`). Using an `Union` means that each caller has to use a cast or instance check on the return value of `parse_multipart()`.